ozaria/site/styles/play/play-level-view.sass

```diff
@@ -8,6 +8,10 @@ body.is-playing
 $level-resize-transition-time: 0.5s
 // TODO: use this for real-time playback but not cinematic playback
 
+#canvas-wrapper
+  top: 0 !important
```

Comment:
Could we avoid the `!important` by making this selector more specific?
`.game-container .level-content #game-area #canvas-wrapper`